src/components/NavigationBar/index.tsx

```diff
@@ -13,13 +13,9 @@ const navLinks = [
 
 const NavigationBar: React.FC = () => {
   const [menuState, setMenuState] = useState(false);
-  const toggleMenu = () => {
-    setMenuState(!menuState);
-  };
-  const scrollToContacts = (): void => {
-    window.scrollTo({ top: document.body.scrollHeight });
-    setMenuState(false);
-  };
```

Comment:
seeking to learn: why did you remove `scrollToContacts`

Reply:
the 2 things happening here are manually forcing the page to scroll and closing the mobile nav menu. we no longer need to force the page to scroll because the link href works as intended by giving the footer an id. closing the mobile nav menu is handled by toggleMenu because the only time it gets called is when menuState is true